README.md

```diff
@@ -47,10 +47,10 @@ Once installation is completed, you can proceed to the first use with a demo ser
 
 If you want to test with Aspera on Cloud, jump to section: [Wizard](#aocwizard)
 
-If you want to test with Aspera demo transfer server:
+If you want to test with Aspera demo transfer server, a default configuration is created on first use:
 
 ```
-$ acli server browse / --url=ssh://demo.asperasoft.com:33001 --username=asperaweb --password=demoaspera
+$ acli server browse /
```

Comment:
This didn't work for me out of the box. Maybe we can instead use dummy arguments for `--url` `--username` and `--password`?

Reply:
I did not change change anything actually, I just re-generated the doc from the `erb.md` file. I'm guessing @laurent-martin is refactoring things and hadn't yet pushed the updated README.md...